src/pages/[platform]/build-a-backend/existing-resources/cli/index.mdx

```diff
@@ -27,8 +27,33 @@ export function getStaticProps(context) {
   };
 }
 
-This guide shows you how to connect new mobile and web apps to AWS resources you've already created using the AWS Amplify CLI (Command Line Interface). The Amplify CLI provides a simple workflow for provisioning cloud resources like authentication, databases, and storage for mobile and web apps through the command line. By following the [first part](/[platform]/build-a-backend/existing-resources/cli/#connect-mobile-app-to-existing-aws-resources) of the guide, you will learn how to connect a new Flutter mobile app to pre-existing backend resources set up through Amplify. The [second part](/[platform]/build-a-backend/existing-resources/cli/#connect-a-web-app-to-existing-aws-resources) of the guide covers how to do the same for a React web app. 
+The AWS Amplify CLI (Command Line Interface) CLI provides a simple workflow for provisioning cloud resources like authentication, databases, and storage for apps through the command line.
 
+<InlineFilter filters={["flutter"]}>
+In this guide, you will learn how to connect a new Flutter mobile app to backend resources you've already created using the Amplify CLI.
+</InlineFilter> 
+
+<InlineFilter filters={["flutter"]}>
+<Callout info>
+
+Connecting a web app? We also offer a version of this guide for integrating existing backends with React using the Amplify CLI. Check out the [React guide](/react/build-a-backend/existing-resources/cli/).
+
+</Callout>
+</InlineFilter> 
+
+<InlineFilter filters={["angular", "javascript", "nextjs", "react", "react-native", "vue"]}>
```

Comment:
@hibler13 same here - the filters here can be combined since the same - 44 and 48